applications/services/desktop/views/desktop_view_pin_input.c

```diff
@@ -6,14 +6,16 @@
 #include <stdint.h>
 
 #include "desktop_view_pin_input.h"
-#include <desktop/desktop_settings.h>
 
 #define NO_ACTIVITY_TIMEOUT 15000
 
 #define PIN_CELL_WIDTH 13
 #define DEFAULT_PIN_X 64
 #define DEFAULT_PIN_Y 32
 
+#define MIN_PIN_LENGTH 4
+#define MAX_PIN_LENGTH 10
```

Comment:
we also have `#define DESKTOP_PIN_CODE_MAX_LEN (14)`

Reply:
Made it 10 everywhere for consistency (but 14 is the threoretical maximum that can be packed into a 32-bit value)